sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusStore.scala

```diff
@@ -146,4 +146,5 @@ class SparkPlanGraphNodeWrapper(
 case class SQLPlanMetric(
     name: String,
     accumulatorId: Long,
-    metricType: String)
+    metricType: String,
+    aggregateMethod: (Array[Long], Array[Long]) => String)
```

Comment:
You could add a default for aggregateMethod by creating a companion object for the case class:
```
object SQLPlanMetric {
  def apply(name: String, accumulatorId: Long, metricType: String): SQLPlanMetric =
    this(name, accumulatorId, metricType, SQLMetrics.stringValue(metricType, _, _))
}
```
Then you don't need to modify existing tests that do `SQLPlanMetric(name, accumulatorId, metricType)`.